velox/functions/sparksql/String.h

```diff
@@ -1130,4 +1130,52 @@ struct ReplaceFunction {
   }
 };
 
+template <typename T>
+struct FindInSetFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<int32_t>& result,
+      const arg_type<Varchar>& str,
+      const arg_type<Varchar>& strArray) {
+    int32_t index = 1;
+    int32_t lastComma = -1;
+
+    if (std::string_view(str).find(',') != std::string::npos) {
+      result = 0;
+      return;
+    }
+
+    auto arrayData = strArray.data();
+    size_t arraySize = strArray.size();
+    size_t matchSize = str.size();
+
+    for (int i = 0; i < arraySize; i++) {
+      if (arrayData[i] == ',') {
+        if (i - (lastComma + 1) == matchSize &&
+            str ==
+                StringView(
+                    strArray.data() + (lastComma + 1), i - (lastComma + 1))) {
```

Comment:
strArray.data() -> arrayData
i - (lastComma + 1) -> matchSize

Reply:
I will fix it. Thanks！